src/components/CardExpirationSection/CardExpirationSection.tsx

```diff
@@ -0,0 +1,62 @@
+import styles from './CardExpirationSection.module.css';
+import { InputSection } from '../InputSection/InputSection';
+import { Dispatch, SetStateAction, useState } from 'react';
+
+type Props = {
+  expiration: string[];
+  setExpiration: Dispatch<SetStateAction<string[]>>;
+};
+
+export default function CardExpirationSection({ expiration, setExpiration }: Props) {
+  const [expirationError, setExpirationError] = useState<string[]>(['', '']);
+
+  const handleExpirationChange = (index: number, value: string) => {
+    let errorMsg = '';
+    if (!/^[0-9]*$/.test(value)) {
+      errorMsg = '숫자만 입력 가능합니다.';
+      return;
+    }
+
+    if (index === 0) {
+      if (value !== '') {
+        const month = Number(value);
+        if (month < 1 || month > 12) {
+          errorMsg = '1부터 12 사이의 숫자를 입력해주세요.';
+        }
+      }
+    }
+    if (index === 1) {
+      if (value !== '' && value.length !== 2) {
+        errorMsg = '2자리 숫자를 입력해주세요.';
+      }
+    }
+
+    const updatedExpiration = [...expiration];
+    updatedExpiration[index] = value;
+    setExpiration(updatedExpiration);
+
+    const updatedError = [...expirationError];
+    updatedError[index] = errorMsg;
+    setExpirationError(updatedError);
+  };
+
+  return (
+    <div className={styles.sectionContainer}>
+      <InputSection.TitleWrapper>
+        <InputSection.Title title="카드 유효기간을 입력해 주세요" />
+        <InputSection.SubTitle title="월/년도(MMYY)를 순서대로 입력해 주세요." />
+      </InputSection.TitleWrapper>
+      <div className={styles.inputSection}>
+        <InputSection.Label text="유효기간" />
+        <InputSection.InputWrapper
+          numbers={expiration}
+          onChange={handleExpirationChange}
+          valid={expirationError.map((msg) => msg === '')}
+          placeholders={['MM', 'YY']}
+          maxLength={2}
+        />
```

Comment:
```suggestion
        <InputSection.InputWrapper
          numbers={expiration}
          onChange={handleExpirationChange}
          valid={expirationError.map((msg) => msg === '')}
          placeholders={['MM', 'YY']}
          maxLength={[2, 2]}
        />
```
`InputWrapper` 컴포넌트는 여러 개의 `Input` 컴포넌트를 생성 할 목적으로 보이네요!

`placeholders` 속성은 각각의 인풋에 placeholder를 설정할 수 있게 잘 되어 있는 것 같은데, `maxLength` 속성은 하나만 받는게 약간 어색한 것 같아요..!

만약 인풋을 2개 생성하는데 `maxLength` 속성이 다를 수도 있으니깐 `placeholders`처럼 각각의 인풋 들어갈 수 있게 설정해주어 유연하게 동작하고 일관성도 지키는 방식은 어떨까요??